src/test/java/com/limechain/runtime/babeapi/scale/BabeApiConfigurationReaderTest.java

```diff
@@ -0,0 +1,35 @@
+package com.limechain.runtime.babeapi.scale;
+
+import com.limechain.babe.api.scale.BabeApiConfigurationReader;
+import com.limechain.chain.lightsyncstate.BabeEpoch;
+import com.limechain.babe.api.BabeApiConfiguration;
+import com.limechain.utils.StringUtils;
+import com.limechain.utils.scale.ScaleUtils;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class BabeApiConfigurationReaderTest {
+    private static final String BABE_API_CONFIGURATION_HEX = "0x701700000000000060090000000000000100000000000000040000000000000018fa3437" +
```

Comment:
Is this buggy for me or we have multiple empty lines?

Reply:
it is real configuration response converted to string hex. I dived it to parts for better readability